articles/connections/references/apple-native/guides/add-siwa-to-web-app.md

```diff
@@ -0,0 +1,230 @@
+      done(null, user);
+    });
+
+    app.get("/.well-known/apple-developer-domain-association.txt", (req, res) => {
+      res.send(domainAssociation);
+    });
+
+    app.get("/auth/apple", passport.authenticate(appleStrategy));
+
+    app.get(
+      "/callback",
+      passport.authenticate(appleStrategy, {
+        successRedirect: "/",
+        failureRedirect: "/login"
+      })
+    );
+
+    app.get("/profile", (req, res) => {
+      res.send(
+        jwt.decode(req.session.passport.user.payload.id_token, { complete: true })
+      );
+    });
+
+    app.get("/", (req, res) => {
+      console.log("User", req.user);
+      res.send(JSON.stringify({ Hello: "World" }));
+    });
+
+    app.listen(port, () => {
+      console.log("Apple Login POC listening on port " + port + "!");
+    });
+    ```
+
+    In the code above, you will notice that there is a reference to the `apple-developer-domain-association.txt` file. Make this file available to the app while running it in your server so Apple can check that you are the owner of the domain you used in the previous section. Move the file you downloaded from Apple to the project root.
+
+3. To confirm that the code is working, execute the following commands locally.
+
+    ``` text
+    export CLIENT_ID=test
+    export CLIENT_SECRET=test
+    export CALLBACK=test
+
+    npm start
+    ```
+
+::: note
+You can leave the test values on the environment variables for the moment. You will configure these variables later.
+:::
+
+If everything works as expected, you will see the contents of the domain association file at the following link: `http://localhost:3000/.well-known/apple-developer-domain-association.txt`. If that is the case, you are ready to make your application run on the internet-accessible server.
+
+### Verify domain ownership on Apple
+
+1. Move your project code to your server and make it run like you did locally. You still don't need to provide the final environment variables. To achieve that, you can use whatever means you prefer. For example, you can use Git, or you can move the files manually (with the help of `scp` or similar). 
+
+2. After running this project on an internet-accessible server (which must respond on behalf of the domain you configured in the Apple developer account), go back to the page you left open (**Register a Services ID**), and click **Verify**. If you got everything right, Apple will confirm that you own the informed domain.
+
+### Configure email relay service
+
+1. To use the email relay service that Apple provides, click **More** under the **Certificates, Identifiers, & Profiles** section and click **Configure**. 
+
+2. Add your domain (`<YOUR CUSTOM DOMAIN>.com`) in the field available on **Domains and Associated Email Addresses** 
+
+3. Click **Register**. 
+
+4. Add an email address (for example, `me@<YOUR CUSTOM DOMAIN>.com`) in **Individual Email Addresses** and click  **Register**.
+
+### Generate client secret
```

Comment:
May want to re-think this title.  Instructions do not cover how to generate the client secret, but rather information about client configuration.